.github/workflows/android-smoke-test.yml

```diff
@@ -7,19 +7,16 @@ on:
       api-level:
         required: true
         type: string
-      try:
-        required: true
-        type: number
     # Map the workflow outputs to job outputs
     outputs:
-      outcome:
-        description:
-        value: ${{ jobs.run.outputs.outcome }}
+      status:
+        description: "Smoke test status"
+        value: ${{ jobs.run.outputs.status }}
 
 jobs:
   run:
-    name: try-${{ inputs.try }}
-    runs-on: macos-latest
+    name: Android Smoke Test
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest-4-cores
```

Reply:
Worth trying this, see if things get faster.
Might just break because the VM images are not the same